### DISCUSSION TITLE
404 page returns status 200!

### DISCUSSION THREAD

**Discussion by ivankorobchuk123** [quasarframework/quasar#16298] [CLI - SSR mode] [answered]
How can I return  notFound.vue component when ssr and status should be 404? 
I found only ` 
if (err.code === 404) {
          res.status(404).send('404 | Page Not Found')
        } `
        but I want my component to be loaded with all styles etc.

**Comment by rstoenescu** [comment, marked as answer]
We have it explained in the docs for [q/app-vite](https://quasar.dev/quasar-cli-vite/developing-ssr/handling-404-and-500-errors) and [q/app-webpack](https://quasar.dev/quasar-cli-webpack/developing-ssr/handling-404-and-500-errors).
